src/components/KMultiselect/KMultiselect.vue

```diff
@@ -1078,7 +1109,7 @@ onBeforeUnmount(() => {
       }
 
       &.input-placeholder-dark::placeholder {
-        color: var(--KInputColor, var(--black-70, rgba(0, 0, 0, 0.7))) !important;
+        color: var(--KInputColor, var(--black-70, var(--kui-color-text, $kui-space-40))) !important;
```

Comment:
issue: this space token shouldn't be here

Reply:
Oops, good catch, fixed